services/QuillLMS/app/controllers/cms/district_admins_controller.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+class Cms::DistrictAdminsController < Cms::CmsController
+  before_action :get_district, only: [:create]
+  before_action :set_district_admin, only: [:destroy]
+
+  def create
+    @district_admin = @district.district_admins.build({user_id: User.find_by(email: district_admin_params)&.id})
```

Comment:
I should add a cop here, but the braces, but a Rails convention is to omit braces for hash parameters if it is the last argument in a method call.  Compare with the `redirect_back` calls in lines 17, 23